kwave/utils/mapgen.py

```diff
@@ -367,11 +367,11 @@ def power_law_kramers_kronig(w: np.ndarray, w0: float, c0: float, a0: float, y:
     return c_kk
 
 
-def water_absorption(f, temp):
+def water_absorption(f: float, temp: Union[float, NDArray[Any,Float]]) -> Union[float, NDArray[Any,Float]]:
```

Comment:
Is a union with an `Any` type helpful? Why do we need `Any`? If we need `Any` Type couldn't we use only the `Any` Type?

Reply:
Sorry I missed that. Any shape makes sense to me.

Reply:
Specifying each shape separately will be verbose. Maybe use pre-defined shapes from `kt` here?